pkg/agent/openflow/cookie/allocator.go

```diff
@@ -102,8 +103,10 @@ func (i ID) String() string {
 
 // Allocator defines operations of a cookie ID allocator.
 type Allocator interface {
-	// Request cookie IDs of flow categories.
+	// Request a cookie IDs of the flow category.
 	Request(cat Category) ID
+	// Request a cookie ID of the flow category and objectID.
+	RequestMask(cat Category, objectID uint32) ID
```

Comment:
The method name

Reply:
Done.

Reply:
I mean the method name `RequestMask` doesn't match its usage. Maybe `RequestWithObjectID`?

Reply:
Got it, updated.